### PR TITLE
chore(absence): Add capability

### PR DESCRIPTION
**Pull request by SystemKeeper**
* Related: https://github.com/nextcloud/talk-ios/pull/1890

## Summary

We started using the absence / out-of-office api in talk-ios recently, but there's no capability for us to check if a server supports the absence api (NC 28) and setting a replacement (NC 30).

## TODO

- [ ] Trigger backports
- [ ] Update documentation to include the capabilities

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by provokateurin — APPROVED** [review]

File: apps/dav/lib/Capabilities.php [hunk 1]
```diff
@@ -7,15 +7,17 @@
 
 use OCP\Capabilities\ICapability;
 use OCP\IConfig;
+use OCP\User\IAvailabilityCoordinator;
 
 class Capabilities implements ICapability {
 	public function __construct(
 		private IConfig $config,
+		private IAvailabilityCoordinator $coordinator,
```
Comment (nickvergessen) [inline apps/dav/lib/Capabilities.php:15]:
```
 There were 2 errors:

1) OCA\DAV\Tests\unit\CapabilitiesTest::testGetCapabilities
ArgumentCountError: Too few arguments to function OCA\DAV\Capabilities::__construct(), 1 passed in /home/runner/actions-runner/_work/server/server/apps/dav/tests/unit/CapabilitiesTest.php on line 22 and exactly 2 expected

/home/runner/actions-runner/_work/server/server/apps/dav/lib/Capabilities.php:13
/home/runner/actions-runner/_work/server/server/apps/dav/tests/unit/CapabilitiesTest.php:22

2) OCA\DAV\Tests\unit\CapabilitiesTest::testGetCapabilitiesWithBulkUpload
ArgumentCountError: Too few arguments to function OCA\DAV\Capabilities::__construct(), 1 passed in /home/runner/actions-runner/_work/server/server/apps/dav/tests/unit/CapabilitiesTest.php on line 37 and exactly 2 expected

/home/runner/actions-runner/_work/server/server/apps/dav/lib/Capabilities.php:13
/home/runner/actions-runner/_work/server/server/apps/dav/tests/unit/CapabilitiesTest.php:37

--

There was 1 failure:

1) OCA\DAV\Tests\unit\CalDAV\Reminder\NotificationProviderManagerTest::testRegisterBadProvider
Your test case is not allowed to access the database.

```

Want a PHP dev to take over?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: apps/dav/lib/Capabilities.php (hunk 1 above)
Comment (SystemKeeper) [reply apps/dav/lib/Capabilities.php:15]: > Want a PHP dev to take over?

Sorry for the trouble. Should be fine now?!

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**